keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifact.kt

```diff
@@ -14,9 +15,11 @@ data class DebianArtifact(
   override val name: String,
   override val deliveryConfigName: String? = null,
   override val reference: String = name,
-  val vmOptions: VirtualMachineOptions,
   override val statuses: Set<ArtifactStatus> = emptySet(),
-  override val from: ArtifactOriginFilterSpec? = null
+  val vmOptions: VirtualMachineOptions,
+  val branch: String? = null,
```

Comment:
Convenience property to indirectly set the `ArtifactOriginFilter` below without typing a bunch of stuff.